api/api/common/config/constants.py

```diff
@@ -39,3 +40,8 @@
 
 FIRST_SCHEMA_VERSION_NUMBER = 1
 SCHEMA_VERSION_INCREMENT = 1
+
+CUSTOM_USER_NAME_REGEX_ENVIRON = os.environ.get("CUSTOM_USER_NAME_REGEX")
```

Comment:
For this we can use the default Python getenv that returns None if the environment variable doesn't exist by default

https://docs.python.org/3/library/os.html#os.getenv

Reply:
Apologies, this code was a holdover from when I was fixing the actions. Removed and changed to .getenv